server/admins/utils.py

```diff
@@ -144,6 +144,36 @@ def delete_music_data(request, **kwargs) -> response.JsonResponse:
         )
 
 
+def delete_comments(request, **kwargs) -> response.JsonResponse:
+    """Deletes exact comment specified by its ID
+
+    Args:
+        request
+        **kwargs
+
+    Returns:
+        response.JsonResponse
+    """
+    try:
+        comment = request.query_params.get("comment")
```

Comment:
Since you are expecting a query parameter, have you defined a URL to accept the delete_comment request? Django works on a model-template view basis so you will need to create a view for deleting comments in the views.py and add a URL path for it as well.